src/components/htmlvideoplayer/plugin.js

```diff
@@ -147,99 +161,100 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
                 return '';
             }
 
-            var defaultAttribute = mediaSource.DefaultSubtitleStreamIndex === t.Index ? ' default' : '';
+            const defaultAttribute = mediaSource.DefaultSubtitleStreamIndex === t.Index ? " default" : "";
 
-            var language = t.Language || 'und';
-            var label = t.Language || 'und';
+            const language = t.Language || "und";
+            const label = t.Language || "und";
             return '<track id="textTrack' + t.Index + '" label="' + label + '" kind="subtitles" src="' + getTextTrackUrl(t, item) + '" srclang="' + language + '"' + defaultAttribute + '></track>';
 
         }).join('');
     }
 
     function getDefaultProfile() {
+        return import("browserdeviceprofile").then(profileBuilder => {
 
-        return new Promise(function (resolve, reject) {
-
-            require(['browserdeviceprofile'], function (profileBuilder) {
-
-                resolve(profileBuilder({}));
-            });
+            return profileBuilder({});
         });
     }
 
-    function HtmlVideoPlayer() {
-
-        if (browser.edgeUwp) {
-            this.name = 'Windows Video Player';
-        } else {
-            this.name = 'Html Video Player';
-        }
-
-        this.type = 'mediaplayer';
-        this.id = 'htmlvideoplayer';
-
-        // Let any players created by plugins take priority
-        this.priority = 1;
-
-        var videoDialog;
-
-        var winJsPlaybackItem;
-
-        var subtitleTrackIndexToSetOnPlaying;
-        var audioTrackIndexToSetOnPlaying;
-
-        var lastCustomTrackMs = 0;
-        var currentClock;
-        var currentSubtitlesOctopus;
-        var currentAssRenderer;
-        var customTrackIndex = -1;
-
-        var showTrackOffset;
-        var currentTrackOffset;
+    /**
+     * Private:
```

Comment:
Add a [private tag](https://jsdoc.app/tags-private.html) to JSDoc comments instead of doing this.

Reply:
That sady doesn't work because I could only declare the whole class private as fast as I figured, it'll be better with ts

Reply:
I'm figuring out a way to use the Class Properties and Private Class Members proposals in code (both supported by Babel, but ESLint is tripping on them with our current config)

Reply:
Well yeah that why I left them out for now, once it works they can easily be declared as such